db/db_tunables.h

```diff
@@ -775,7 +775,7 @@ REGISTER_TUNABLE("no_static_tag_blob_fix", NULL, TUNABLE_BOOLEAN,
                  &gbl_force_notnull_static_tag_blobs,
                  INVERSE_VALUE | READONLY | NOARG, NULL, NULL, NULL, NULL);
 REGISTER_TUNABLE("nosurprise", NULL, TUNABLE_BOOLEAN, &gbl_surprise,
-                 INVERSE_VALUE | READONLY | NOARG, NULL, NULL, NULL, NULL);
+                 READONLY | NOARG, NULL, NULL, NULL, NULL);
```

Comment:
I think we need the INVERSE_VALUE flag here, "nosurprise" ==> !"surprise" ==> !gbl_surprise